### PR TITLE
Add AgoraNodes EVMOS RPC, gRPC, JSON-RPC, API

### PR DESCRIPTION
**Pull request by AgoraNodes**

### CODE REVIEW

File: evmos/chain.json [hunk 1]
```diff
@@ -107,6 +107,11 @@
         "id": "d4a5fcc56f369665c232788c22844dc3a018fe08",
         "address": "peer.evmos.silknodes.io:26602",
         "provider": "Silk Nodes"
+      }
```
Comment (JeremyParish69) [inline evmos/chain.json:110]: need comma here (for each one you added)

File: evmos/chain.json [hunk 2]
```diff
@@ -171,6 +176,10 @@
       {
         "address": "https://rpc.evmos.silknodes.io/",
         "provider": "Silk Nodes"
+      }
```
Comment (JeremyParish69) [inline evmos/chain.json:179]: still missing comma here

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
still need comma for rpc and peer

**Comment by AgoraNodes** [comment]
added commas should be good to go

**Pull request merged**